src/FindFund/FindFund.js

```diff
@@ -36,12 +37,14 @@ const columnMapping = {
 };
 const resultsFormatter = {};
 
+const INIT_PAGINATION = { limit: PLUGIN_RESULT_COUNT_INCREMENT || 35, offset: 0 };
```

Comment:
just curious why `|| 35`

Reply:
Oh, it was just for testing, I'll remove it, thx)!